rclpy/src/rclpy/_rclpy.c

```diff
@@ -77,6 +77,66 @@ rclpy_get_sigint_guard_condition(PyObject * Py_UNUSED(self), PyObject * Py_UNUSE
   return pylist;
 }
 
+/// Create a general purpose guard condition
+/**
+ * A successful call will return a list with two elements:
+ *
+ * - a Capsule with the pointer of the created rcl_guard_condition_t * structure
+ * - an integer representing the memory address of the rcl_guard_condition_t
+ *
+ * \return a list with the capsule and memory location, or
+ * \return NULL on failure
+ */
```

Comment:
The doc block should mention what function should be called to destroy the created enitty.

Reply:
ec4bf8e5da4d1eaca06a856eda2a14b6bf70f0d6